.github/workflows/release-binaries.yml

```
@@ -43,3 +43,20 @@ jobs:
       rust_log: info
     # This step needs access to Docker Hub secrets to run successfully
     secrets: inherit
+
+  failure-issue:
+    name: Open or update issues for release binaries failures
+    # When a new job is added to this workflow, add it to this list.
+    needs: [ build ]
```

I think we want to fail if our experimental image fails, because building is reliable. It's only running on testnet that is unreliable, and the image build doesn't do that.

```suggestion
    needs: [ build, build-mining-testnet ]
```